src/test/color-input.test.ts

```diff
@@ -154,7 +166,30 @@ function createColorInputTest(
     const value = config[ColorSettings.titleBar_activeBackground];
     stub.restore();
 
-    // undefined is OK, since that measn they hit ESC or blank
+    // undefined is OK, since that means they hit ESC or blank
+    // Otherwise, we need a valid color
+    assert.ok(!value || isValidColorInput(value));
+    assert.equal(expectedValue, value);
+  };
+}
+
+function createColorInputTestWithParam(
```

Comment:
maybe you should split this into 2 functions so the logic is less complicated.

Reply:
Done. And used `assert.rejects` a better version of testing promise that throws an error without using try/catch. https://nodejs.org/api/assert.html#assert_assert_rejects_asyncfn_error_message